srcpkgs/snore/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'snore'
+pkgname=snore
+version=0.2
+revision=1
+build_style=gnu-makefile
+short_desc="Sleep with feedback"
+maintainer="wael <40663@proton.me>"
+license="MIT"
+homepage="https://github.com/clamiax/snore"
+distfiles="https://github.com/clamiax/$pkgname/archive/refs/tags/${version}.tar.gz"
+checksum=2b160061589e54b6a88b2995028491b38f9695393bf48af5ad61835fb6fd771a
+CFLAGS="-DVERSION=\\\"${version}\\\""
+
+post_install() {
+	vlicense LICENSE
+	vdoc README.md
```

Comment:
@Chocimier  ??? i have added it?

Reply:
https://github.com/void-linux/void-packages/pull/37299#discussion_r882847673

Reply:
i had the changes there but i forgot to push...